use-cases/grpc_python/Makefile

```
@@ -26,3 +29,25 @@ run:
 
 down:
 	PIPELINE_COUNT=$(PIPELINE_COUNT) MODEL_NAME=$(MODEL_NAME) RETAIL_USE_CASE_ROOT="$(RETAIL_USE_CASE_ROOT)" RESULTS_DIR="$(RESULTS_DIR)" docker compose -f $(DOCKER_COMPOSE) down
+
+build-minikube:
+	minikube start
+	minikube image build -t grpc_python:dev -f Dockerfile .
+	minikube image build -t ovms-server:dev -f use-cases/grpc_python/pipeline-server/Dockerfile ../../
+
+run-minikube-demo: build-minikube
+	kompose -f docker-compose_grpc_python.yml convert -o kubernetes/ && \
+	kubectl apply -f kubernetes
+	
+stop-minikube-demo:
+	kubectl delete -f kubernetes
+	minikube stop
+
+helm-convert:
+	kompose -f docker-compose_grpc_python.yml convert -o object-detection-service-distributed-grpc -c
+
+install-helm:
+	helm install -f kubernetes/Chart.yaml dls kubernetes
```

the Chart.yaml is under the object-detection-service-distributed-grpc folder not kubernetes
```suggestion
	helm install -f object-detection-service-distributed-grpc/Chart.yaml dls kubernetes
```